android/src/main/java/com/robinpowered/react/vitals/RNVitalsModule.java

```diff
@@ -45,6 +50,9 @@ public String getName() {
   public @Nullable Map<String, Object> getConstants() {
     HashMap<String, Object> constants = new HashMap<String, Object>();
     constants.put("LOW_MEMORY", LOW_MEMORY);
+    constants.put("MEMORY_CRITICAL", MEMORY_CRITICAL);
+    constants.put("MEMORY_LOW", MEMORY_LOW);
+    constants.put("MEMORY_CRITICAL", MEMORY_CRITICAL);
```

Comment:
If we're keeping a group of items flat with other arbitrary items, it should be namespaced as much as possible (`MEMORY_LEVEL_XYZ`).

But ideally, it would be grouped (`Vitals.MemoryLevel.CRITICAL`).

Let's group that 1 level deeper.